hm03_ortizg.py

```diff
@@ -0,0 +1,166 @@
+#################################################################################
+# Author: Giovanny Ortiz
+# Username: ortizg
+#
+# Assignment: HW03: "Fully Functional Gitty Psychedelic Robotic Turtles"
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1OJ2Gq9T8-_tipjT83M_Q2aLOkE_8c5aM-uYcQoyR1oc/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# ChatGPT for references, methods and ways to code. (No copy paste obviously)
+#
+#################################################################################
+
+import turtle
+
+
+def tierra_ciel(t):
+    """This function makes the green surface on the bottom of the screen simulating the ground"""
+    t.speed(5)
+    t.penup()
+    t.setposition(-385, -80)
+    t.pendown()
+    for i in range(4):
+        t.forward(900)
+        t.right(90)
+        t.forward(30)
+        t.right(90)
+        t.forward(900)
+        t.left(90)
+        t.forward(30)
+        t.left(90)
+    t.forward(900)
+    t.penup()
+    t.home()
+    t.hideturtle()
+    pass
+    # ....
```

Comment:
Remove the placeholder text from the template, it was used to prevent a runtime error.